src/main/java/ch/wisv/areafiftylan/service/TicketService.java

```diff
@@ -13,6 +15,8 @@
 
     Integer getNumberSoldOfType(TicketType type);
 
+    Collection<Ticket> findValidTicketsByOwnerUsername(String username);
```

Comment:
The UserService is the only class with access to the Repository. I think it's better to leave all the User logic to the Service. The username is something in the Spring Security auth object and therefore doesn't require any logic done by the controller. 

Most Service functions work references(such asid's or usernames) to prevent User objects floating through the entire application. This is safer from a persistence perspective.